source/documentation/administration_guide/topics/Renaming_the_Manager_with_the_Ovirt_Engine_Rename_Tool.adoc

```diff
@@ -55,3 +55,28 @@ This command modifies the FQDN in each self-hosted engine node's local copy of [
 This command modifies the FQDN in the main copy of [filename]`/etc/ovirt-hosted-engine-ha/hosted-engine.conf` on the shared storage domain.
 
 Now, all new and existing self-hosted engine nodes use the new FQDN.
+
+
+[NOTE]
+====
+The oVirt Engine Rename Tool is designed to work only on local machines. Changing the {engine-name} name does not automatically update the name on remote Data Warehouse machines. Changing the names on remote DWH machines must be performed manually.
+====
+
+*For remote Data Warehouse deployments, follow these steps on the remote machine (not on the {engine-name} machine):*
+
+. Remove the following PKI files:
++
+`/etc/pki/ovirt-engine/apache-ca.pem`
+`/etc/pki/ovirt-engine/apache-grafana-ca.pem`
+`/etc/pki/ovirt-engine/certs/\*`
+`/etc/pki/ovirt-engine/keys/*`
+. In the following files, update the engine fqdn to the new name (for example, `vm-new-name.local_lab_server.redhat.com`):
++
+`/etc/grafana/grafana.ini`
+`/etc/ovirt-engine-dwh/ovirt-engine-dwhd.conf.d/10-setup-database.conf`
+`/etc/ovirt-engine-setup.conf.d/20-setup-ovirt-post.conf`
+. Run engine-setup:
++
+----
+# engine-setup
```

Comment:
Perhaps mention that this will try to update, if an update is available - and that `--offline` can be used to prevent that.